phy_tester/test_eth_phy.py

```diff
@@ -327,7 +327,7 @@ def ethernet_phy_test(dut: Dut, test_pc_nic: str) -> None:
     print(f'loopback server: {tx_path}{res_dict["loopback server"]}{norm}')
     draw_result(tx_path, rx_path, tx_path, rx_path, False, True)
 
-    if True is True: #res_dict['loopback server'] == 'FAIL':
```

Comment:
This worked before?

Reply:
Yes, it worked. The workflow of PHY tester is to test the full loop `PC-PHY-EMAC-DUT-EMAC-PHY-PC` at first. If it passes then there is no need to run the isolated tests. This condition was there to force to run the subtests everytime which was good for my development.

Reply:
should we add a config option to "run all tests unconditionally"?